doc/dcTutorialCpp.md

```diff
@@ -1,29 +1,36 @@
 
-# DynamicsComputations tutorial
+# KinDynComputations tutorial
 
-To use the DynamicsComputations class, you have to include its header.
+To use the KinDynComputations class, you have to include its header.
 For this tutorial we also use the appropriate `using` definitions to 
-avoid having to type the namespace of the class. 
+avoid having to type the `iDynTree` namespace.
 
 ~~~cpp
 #include <iostream>
 
-#include <iDynTree/HighLevel/DynamicsComputations.h>
+#include <iDynTree/KinDynComputations.h>
+#include <iDynTree/ModelIO/ModelLoader.h>
 
 using namespace iDynTree;
-using namespace iDynTree::HighLevel;
 ~~~
 
 The first thing that we have to do 
 is to initialize the class by providing a model of the robot. Currently 
 iDynTree supports only URDF models to specify the model. 
+All the logic for loading models, including loading reduced models,
+is contained in the iDynTree::ModelLoader class.
 ~~~cpp
 int main()
 {
-    DynamicsComputations dynComp;
-    dynComp.loadRobotModelFromFile("model.urdf");
-    std::cout << "The loaded model has " << dynComp.getNrOfDegreesOfFreedom() 
-          << "internal degrees of freedom and " << dynComp.getNrOfLinks() 
+    ModelLoader mdlLoader;
+    bool ok = mdlLoader.loadModelFromFile(urdf_filename);
+
+    // Check that ok is actually true
+
+    KinDynComputations kinDynComp;
+    kinDynComp.loadRobotModel("model.urdf");
```

Comment:
This should not work! 

`kinDynComp.loadRobotModel(mdlLoader.model());`

Reply:
I have toooo many failed attempts to get good documentation around, I should just thrown them our. :D 
Anyway, nice catch!